.github/workflows/full-ci.yml

```diff
@@ -191,11 +191,22 @@ jobs:
         run: |
           echo "TT_CLI_TARANTOOL_PREFIX=${GITHUB_WORKSPACE}/include/" >> $GITHUB_ENV
 
+      - name: Prepare Docker
+        run: |
+          echo "DOCKER_HOST=unix:///${HOME}/.docker/run/docker.sock" >> $GITHUB_ENV
+          docker context use default
+
       - name: Run unit tests
         run: mage unitfull
 
       - name: Integration tests
-        run: mage integrationfull
+        run: |
+          mkdir ${HOME}/tmp || true
```

Comment:
Why not create files in the macOS temporary directory?

Reply:
By default the `TMPDIR` is locating in `/private/var/` on macOS runner and decrements with long hash which causes problem with long path length in `test_status`. Wrote about this problem [here](https://github.com/tarantool/tt/issues/969).

Now edited on `/tmp` dir instead of `~/tmp` in workflow.